lms/static/sass/_build-lms-v1.scss

```diff
@@ -84,4 +84,4 @@
 
 // POC dashboard
 @import 'poc-dashboard';
-@import 'learner-dashboard';
\ No newline at end of file
+@import 'learner-dashboard';
```

Comment:
@rosheen33 what is the reason for this diff. it should not show up in this changeset.